packages/cms-cli/commands/auth.js

```diff
@@ -22,58 +22,60 @@ const {
 } = require('../lib/usageTracking');
 const { PORTAL_ID, CLIENT_ID, CLIENT_SECRET } = require('../lib/prompts');
 
-const getAuthContext = async () => {
+const getAuthContext = () => {
   const prompt = inquirer.createPromptModule();
   return prompt([PORTAL_ID, CLIENT_ID, CLIENT_SECRET]);
 };
 
 const COMMAND_NAME = 'auth';
 
+async function authAction(type, options) {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+
+  if (!validateConfig()) {
+    process.exit(1);
+  }
+
+  if (type !== 'oauth2') {
+    logger.error("The only supported authentication protocol is 'oauth2'");
+    return;
+  }
+  const answers = await getAuthContext();
```

Comment:
If `getAuthContext()` is not `async`, can we remove `await` here?

Reply:
we still need async here to await the async return from prompt, which is returned by getAuthContext

Reply:
Should `getAuthContext` be `async` then?

Reply:
I realized after my comment that the code had changed, so feel free to ignore.